front/app/containers/Admin/projects/projectFolders/components/ProjectFolderRow/ProjectFolderRow.test.tsx

```diff
@@ -54,16 +103,28 @@ jest.mock('hooks/useAuthUser', () => {
   return () => mockUserData;
 });
 
+// Needed to render folder with project inside
+jest.mock('hooks/useAdminPublications', () => {
+  return () => mockFolderChildAdminPublications;
+});
+
 const props: Props = {
-  publication,
+  publication: folderPublication,
 };
 
 describe('ProjectFolderRow', () => {
+  it('renders the project row inside', () => {
+    render(<ProjectFolderRow {...props} />);
+
+    const projectRows = screen.getAllByTestId('projectRow');
+    expect(projectRows.length).toEqual(1);
+  });
+
   describe('When user is an admin', () => {
     it('shows the edit button', () => {
       render(<ProjectFolderRow {...props} />);
 
-      const editButton = screen.getByRole('button', { name: 'Edit' });
+      const editButton = screen.getByTestId('folder-row-edit-button');
```

Comment:
It wouldn't be a problem after all, because the project row edit button is a link, not a button. But still  good in a component with many buttons to ensure you get the  right button as we discussed.